src/actions/widgets/index.js

```diff
@@ -0,0 +1,5 @@
+export const CLOCK_TICK = 'clockTick'
+
+const clockTick = time => ({ type: CLOCK_TICK, time })
+
+export { clockTick }
```

Comment:
Could you combine exports into one object based on consistency through out this PR?